test/protobuf/decoder_test.exs

```diff
@@ -126,7 +126,7 @@ defmodule Protobuf.Decoder.Test do
       use Protobuf, from: Path.expand("../proto/mumble.proto", __DIR__)
     end
 
-    msg = MumbleProto.Authenticate.new(username: "bitwalker")
+    msg = MumbleProto.MumbleProto.Authenticate.new(username: "bitwalker")
```

Comment:
I feel super confused. I didn't think this PR changed the meaning of `inject` at all. I thought the only difference was that `inject` defined your protobufs from the `Elixir.` instead of `__MODULE__.`, which I think is the exact same behaviour that was there before. Am I misunderstanding something?